src/db.cr

```diff
@@ -113,12 +113,42 @@ module DB
     end
   end
 
+  # Opens a connection using the specified *uri*.
+  # The scheme of the *uri* determines the driver to use.
+  # Returned connection must be closed by `Connection#close`.
+  # If a block is used the connection is yielded and closed automatically.
+  def self.connect(uri : URI | String)
+    build_connection(uri)
+  end
+
+  # Same as `#connect` but the database is yielded and closed automatically.
```

Comment:
Shouldn't this comment be `ditto`? Block behavior is already been mentioned in `self.connect` documentation. :smile: